target/linux/mpc85xx/patches-5.10/108-powerpc-85xx-red-50-rev1.patch

```diff
@@ -0,0 +1,29 @@
+--- a/arch/powerpc/platforms/85xx/Kconfig	2022-06-23 11:33:47.516512534 +0200
```

Comment:
It would be nice to have it in upstream as well with the DTS.

Reply:
I'm a bit afraid of this process :-) If you all think this patchset is good, I will give it a try!